logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/AvsClient.kt

```diff
@@ -0,0 +1,17 @@
+package com.wire.kalium.logic.data.call
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+@Serializable
+data class AvsClient(
```

Comment:
Maybe it would be better if we remove that "Avs" prefix from the name, `CallClient` ? Avs lib is somthing internal of calling module and the logic module shouldn't be aware of it